admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html

```diff
@@ -30,21 +30,10 @@
 <script src="/etc/felibs/admin/dependencies.js"></script>
 <script src="/etc/felibs/admin.js"></script>
 
-<!--extensions, this should be replaced with pulling in the actual extensions defined in the system?
-<script src="/etc/felibs/examplesite/js/exampleExtensionsAdminpagessubnav.js"></script>
 <script>
-    Vue.component('example-extensions-adminpagessubnav', cmpExampleExtensionsAdminpagessubnav)
-</script>
-/extensions-->
-
-<script>
-    const VueFormGenerator = window.VueFormGenerator;
-    const VueMultiselect = window.VueMultiselect
-    const VueImagesLoaded = window.VueImagesLoaded
-    const VueTrumbowyg = window.VueTrumbowyg
     Vue.component('vue-form-generator', VueFormGenerator.component)
     Vue.component('vue-multiselect', VueMultiselect.default)
-    Vue.component('datetime', window.VueDatetime.Datetime);
+    Vue.component('datetime', window.VueDatetime.Datetime)
```

Comment:
If we are removing everything from `window` (which is good) why is VueDatetime still in it and referred here ?